inference-engine/thirdparty/clDNN/src/kernel_selector_helper.cpp

```diff
@@ -832,8 +832,8 @@ void set_params(const program_node& node, kernel_selector::params& params) {
     params.engineInfo.maxImage2dWidth = device_info.max_image2d_width;
     params.engineInfo.maxImage2dHeight = device_info.max_image2d_height;
     params.engineInfo.computeUnitsCount = device_info.execution_units_count;
-    params.engineInfo.maxThreadsPerExecutionUnit = device_info.max_threads_per_execution_unit;
-    params.engineInfo.maxThreadsPerDevice = device_info.max_threads_per_device;
+    params.engineInfo.maxThreadsPerExecutionUnit = device_info.num_threads_per_eu > 0 ? device_info.num_threads_per_eu : 7;
+    params.engineInfo.maxThreadsPerDevice = params.engineInfo.maxThreadsPerDevice * device_info.execution_units_count;
```

Comment:
Did you intend `computeUnitsCount * execution_units_count`?
For our DG2 with 128EU, `maxThreadsPerDevice` is already 896.

Reply:
No, I didn't. It was only a typo, here should be `maxThreadsPerExecutionUnit`. Thanks and fixed.